### PR TITLE
feat: Cleanup doc site

### PR DESCRIPTION
**Pull request by DaveSkender**
## Summary

Cleanup and refinements to the v3 documentation site, primarily focused on the `IndicatorChart.vue` chart component.

### Changes

**LWC v5 API migration**
- Replace deprecated `series.setMarkers()` with `createSeriesMarkers(series, markers)` module-level function

**Oscillator price context (`subPanel` prop)**
- New `subPanel` prop (default `false`): oscillator indicator pages now display price candlesticks above the oscillator for visual context
- Set `subPanel="true"` on landing page oscillator charts to suppress the price panel

**Marker positioning system**
- New `markerPosition` field in `SeriesStyle` JSON: `'inBar' | 'aboveBar' | 'belowBar' | 'auto' | 'atPrice'`
- `resolveMarkerPosition()` translates config values to LWC v5 marker position API values
- `isGreenColor()` helper for robust pointer direction detection

**Code quality fixes (swarm review)**
- Fixed duplicate `'atPrice'` union member in `chart-types.ts`
- Added explicit `'inBar'` branch (previously fell through to auto-logic)
- Typed `configured` as `SeriesStyle['markerPosition']` instead of `string | undefined`
- Merged identical `case 'dots': case 'pointer':` switch arms
- Extracted shared marker-building block (was duplicated per type)
- `seriesMarkersPlugin.detach()` called before null in `destroyChart()`

**JSON data files**
- Added `markerPosition` field to dots-type series in: AtrStop, Fractal, HomeOverlay, ParabolicSar, Pivots, VolatilityStop

### Testing
- All 2433 unit tests pass
- markdownlint: 0 errors
- dotnet build: 0 warnings
- Visual audit: all indicator charts render correctly

### CODE REVIEW

File: .github/workflows/test-website-links.yml [hunk 1]
```diff
@@ -62,5 +62,5 @@ jobs:
           htmlproofer .vitepress/dist
           --no-enforce-https
           --no-check-external-hash
-          --ignore-status-codes "0,302,403,406,408,415,429,503,999"
+          --ignore-status-codes "0,302,402,403,406,408,415,429,503,999"
```
[the remark this answers is not in the document]
Comment (DaveSkender) [reply .github/workflows/test-website-links.yml:65]: these are false flags and something we can’t fix
